python/kserve/kserve/utils/utils.py

```diff
@@ -225,3 +236,69 @@ def strtobool(val: str) -> bool:
         return False
     else:
         raise ValueError("invalid truth value %r" % (val,))
+
+
+def get_grpc_client(predictor_host: str, use_ssl: bool) -> grpc_predict_v2_pb2_grpc.GRPCInferenceServiceStub:
```

Comment:
We can reuse the `inference_client.py`
https://github.com/kserve/kserve/blob/master/python/kserve/kserve/inference_client.py